sources/core/Stride.Core.Reflection/TypeDescriptors/ObjectDescriptor.cs

```diff
@@ -144,7 +142,7 @@ public virtual void Initialize(IComparer<object> keyComparer)
             }
 
             // Free the member list
-            members = memberList.ToArray();
+            members = [.. memberList];
```

Comment:
The new syntax is cool, but maybe this is a bit too much, let's keep `ToArray()`, it is more readable

Reply:
reverted